src/react/workflow/ExpirationForm.tsx

```diff
@@ -335,115 +371,105 @@ export function ExpirationForm({ bucketList, locations, prefix = '' }: Props) {
-              <T.Row data-testid="toggle-action-expire-previous-version">
-                <T.KeyTooltip
-                  tooltipMessage={`
+                  <SpacedBox mr={8} />
+                  <PluralizeDays number={currentVersionTriggerDelayDays} />
+                </div>
+              </T.Value>
+            </T.Row>
+            <T.Row style={{ minHeight: 'initial', height: 'auto' }}>
+              <T.Key size={60}></T.Key>
+              <T.Value>
+                <ErrorInput
+                  id="error-currentVersionTriggerDelayDays"
+                  style={{ height: 'auto' }}
+                  error={
+                    errors[`${prefix}currentVersionTriggerDelayDays`]?.message
+                  }
+                />
+              </T.Value>
+            </T.Row>
+            <T.Row data-testid="toggle-action-expire-previous-version">
+              <T.KeyTooltip
+                tooltipMessage={`
                 All the objects that become previous versions older than ${
                   previousVersionTriggerDelayDays || 'provided'
                 } days will be permanently deleted.
                 `}
-                  tooltipWidth="13rem"
-                  size={60}
-                >
-                  {' '}
-                  Expire <span style={{ fontWeight: 'bold' }}>
-                    Previous
-                  </span>{' '}
-                  version of objects{' '}
-                </T.KeyTooltip>
-                <T.Value
+                tooltipWidth="13rem"
+                size={60}
+              >
+                Expire <span style={{ fontWeight: 'bold' }}>Previous</span>{' '}
+                version of objects
+              </T.KeyTooltip>
+              <T.Value style={{ ...flexStyle }}>
+                <Controller
+                  control={control}
+                  name={`${prefix}previousVersionTriggerDelayDays`}
+                  render={({
+                    field: { onChange, value: previousVersionTriggerDelayDays },
+                  }) => {
+                    return (
+                      <Toggle
+                        id="previousVersionTriggerDelayDaysToggle"
+                        placeholder="previousVersionDelayDaysToggle"
+                        disabled={!isSourceBucketVersionned}
+                        toggle={
+                          previousVersionTriggerDelayDays !== null &&
+                          previousVersionTriggerDelayDays !== undefined
+                        }
+                        onChange={(e) => onChange(e.target.checked ? 7 : null)}
+                      />
+                    );
+                  }}
+                />
+
+                {!isSourceBucketVersionned ? (
+                  <IconHelp
+                    tooltipMessage="This action is disabled when source bucket is not versionned"
+                    variant="outline"
+                    overlayStyle={{ width: '13rem' }}
+                  />
```

Comment:
Would be better to display as a helper text instead of within the tooltip.

Reply:
@Cuervino ?

Reply:
1. I'm not 100% sure, adding too many helper texts might be cumbersome.
2. it should be "versioned", not "versionned".

Reply:
<img width="685" alt="Capture d’écran 2022-10-10 à 18 17 18" src="https://user-images.githubusercontent.com/7314118/194911000-a6df61d6-1e95-4668-98dc-97daa05e8735.png">

We got something like that

Reply:
I prefer the tooltip, here we don't know with what the test is related to.